redhat/rdma-core.spec

```diff
@@ -53,7 +53,7 @@ BuildRequires: python-docutils
 BuildRequires: perl-generators
 %endif
 
-Requires: dracut, kmod, systemd, pciutils
```

Comment:
If you are going to get rid of the requires on what provides /sbin/udevadm then the %post needs to check if /sbin/udevadm exists before trying to run it, right?

Reply:
Yes, you are right. I will squash next commit into this to fix this.